libcxx/test/std/atomics/atomics.types.generic/trivially_copyable.verify.cpp

```diff
@@ -15,7 +15,7 @@
 // is not trivially copyable, however Clang will sometimes emit additional
 // errors while trying to instantiate the rest of std::atomic<T>.
 // We silence those to make the test more robust.
-// ADDITIONAL_COMPILE_FLAGS: -Xclang -verify-ignore-unexpected=error
+// ADDITIONAL_COMPILE_FLAGS(verify-support): -Xclang -verify-ignore-unexpected=error
```

Comment:
I'm somewhat surprised that you're running any `.verify.cpp` tests. I was always under the impression that they are basically useless for anybody but libc++. How exactly are they useful to you?

Reply:
I think we're skipping them - I added this during my initial round of trying to properly guard all of the compile flags, but I didn't check to see whether we actually need this. I'll check now.

Reply:
Yep, great catch, we didn't need this! Reverted.